### PR TITLE
fixing gofmt errors in the files

### PR DESCRIPTION
**Pull request by poothia**
it fixes go-fmt errors in the files.

### CODE REVIEW

**Comment by marun** [comment]
Please remove changes to generated files.

Also, I don't think there is much value in mechanical changes like this one.  If we care about this kind of thing it should be in an automated check.

**Comment by gyliu513** [comment]
@marun the same issue also exist for `cluster-registry`, when I run https://github.com/kubernetes/cluster-registry/blob/master/docs/development.md#verify-go-source-files , some auto generated files will also be updated.

/cc @font may have more comments for this.

**Comment by marun** [comment]
@gyliu513 Those file should be excluded from update.  There is zero value in running govet or gofmt against files that are mechanically maintained.

**Comment by shashidharatd** [comment]
@poothia, please exclude generated files and retain only the changes for non-generated code.

**Comment by poothia** [comment]
@shashidharatd @font @marun please review this again, i have removed the generated files from here, it was my mistake to add them in the first place, because i thought all the files except vendor folder belongs to project.

File: pkg/apis/multiclusterdns/doc.go [hunk 1]
```diff
@@ -15,11 +14,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-
-
```
Comment (shashidharatd) [inline pkg/apis/multiclusterdns/doc.go]: This is generated by kubebuilder. And all the subsequent changes in doc.go. please exclude them.

**Pull request closed without merging**